triage/web_excel_compatibility_rules.py

```
@@ -23,6 +25,40 @@ def _read_text(zf: zipfile.ZipFile, name: str) -> str:
+    if rel_name == "_rels/.rels":
+        return ""
+    if "/_rels/" not in rel_name or not rel_name.endswith(".rels"):
+        return ""
+    prefix, rel_file = rel_name.split("/_rels/", 1)
+    source_part = f"{prefix}/{rel_file[:-5]}"
+    return posixpath.dirname(source_part)
+
+
+def _relationship_targets(zf: zipfile.ZipFile, rel_name: str) -> List[tuple[str, str]]:
+    try:
+        root = ET.fromstring(zf.read(rel_name))
```

Ruff (0.15.21)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)